src/haddock/modules/analysis/caprieval/defaults.yml

```diff
@@ -3,99 +3,99 @@ basic:
     default: ''
     type: path
     length: 9999
-    hoover: No help yet
+    hover: No help yet
```

Comment:
I think `tooltip` is better 😉

Reply:
Solved with `title`, `short` and `long`.